xinference/deploy/test/test_error_message.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from click.testing import CliRunner
+
+from xinference.deploy.cmdline import model_launch
+
+
+def test_model_launch_error():
+    runner = CliRunner()
+    result = runner.invoke(
+        model_launch,
+        [
+            "--model-engine",
+            "transformers",
+            "--model-name",
+            "qwen2.5-instruct",
+            "--model-uid",
+            "-s",
+            "0.5",
+            "-f",
+            "gptq",
+            "-q",
+            "INT4",
+            "111",
+            "-l",
+        ],
+    )
+    print(result.output)
+    assert result.exit_code == 1
+    assert (
+        "You must specify extra kwargs with `--` prefix. There is an error in parameter passing that is 0.5."
+        in str(result)
+    )
+
+
+def test_error_parameters():
```

Comment:
What's the differences for these two tests? IMHO, they can be merged into one.

You need to add  comment to tell others what you are testing.